library/lua/script-manager.lua

```diff
@@ -88,28 +88,38 @@ local INSTALLED_MODS_PATH = 'data/installed_mods/'
 -- changes to the files)
 local MOD_PATH_ROOTS = {WORKSHOP_MODS_PATH, MODS_PATH, INSTALLED_MODS_PATH}
 
-local function get_mod_id_and_version(path)
+function get_mod_info(path)
```

Comment:
This is not the API that you should be exposing. Requiring your clients to know the system path to a mod directory is too low level. Try to keep that knowledge within this file. Can you rework `gui/mod-manager` to ask a higher-level question?